apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/schemas/Sgv2TablesResourceImpl.java

```diff
@@ -123,31 +123,29 @@ public Uni<RestResponse<Sgv2NameResponse>> createTable(
   @Override
   public Uni<RestResponse<Sgv2NameResponse>> updateTable(
       final String keyspaceName, final String tableName, final Sgv2TableAddRequest tableUpdate) {
-    return queryWithTableAsync(
-            keyspaceName,
-            tableName,
-            true,
-            (tableDef) -> {
-              Sgv2Table.TableOptions options = tableUpdate.tableOptions();
-              List<?> clusteringExpressions = options.clusteringExpression();
-              if (clusteringExpressions != null && !clusteringExpressions.isEmpty()) {
-                throw new WebApplicationException(
-                    "Cannot update the clustering order of a table", Status.BAD_REQUEST);
-              }
-              Integer defaultTTL = options.defaultTimeToLive();
-              // 09-Dec-2021, tatu: Seems bit odd but this is the way SGv1/RESTv2 checks it,
-              //    probably since this is the only thing that can actually be changed:
-              if (defaultTTL == null) {
-                throw new WebApplicationException(
-                    "No update provided for defaultTTL", Status.BAD_REQUEST);
-              }
-              return new QueryBuilder()
-                  .alter()
-                  .table(keyspaceName, tableName)
-                  .withDefaultTTL(options.defaultTimeToLive())
-                  .parameters(PARAMETERS_FOR_LOCAL_QUORUM)
-                  .build();
-            })
+    Sgv2Table.TableOptions options = tableUpdate.tableOptions();
+    List<?> clusteringExpressions = options.clusteringExpression();
+    if (clusteringExpressions != null && !clusteringExpressions.isEmpty()) {
+      throw new WebApplicationException(
+          "Cannot update the clustering order of a table", Status.BAD_REQUEST);
```

Comment:
```
return Uni.createFrom().failure(new WebApplicationException(
          "Cannot update the clustering order of a table", Status.BAD_REQUEST));
```

Always prefer this that throwing exceptions in the JVM

Reply:
Ok I can change that.